submissions/Harnytskyi/js-pre-oop/index.js

```diff
@@ -0,0 +1,82 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://github.com/Harnytskyi/a-tiny-JS-world
+   Web app: https://harnytskyi.github.io/a-tiny-JS-world 
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+class Inhabitant {
+    constructor(species, name, gender, legs, hands, saying) {
+        this.species = species;
+        this.name = name;
+        this.gender = gender;
+        this.legs = legs;
+        this.hands = hands;
```

Comment:
Not all inhabitants have hands at all. Not just `0` hands but no hands at all.
Please check extra materials in the task and TJSW project to understand OOP principles better and employ them.